src/main/java/Duke.java

```diff
@@ -1,10 +1,180 @@
+        printLine();
+        System.out.println("      See ya! Don't wanna be ya.");
+        printLine();
+    }
+
+    /**
+     * Prints the list of tasks collated by Tired.
+     */
+    public static void printList() {
+        System.out.println("      Here are your tasks human:");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println("      " + (i + 1) + "." + list[i]);
+        }
+    }
+
+    /**
+     * Adds inputs from user to list[] to keep track of user's tasks, deadlines, and events.
+     *
+     * @param taskName Name of task from user.
+     * @param taskType Type of task from user.
+     * @param details Time/date of event/deadline.
+     */
+    public static void addTask(String taskName, String taskType, String details) {
+        switch (taskType) {
+        case "todo":
+            list[taskCount] = new Todo(taskName);
+            break;
+        case "deadline":
+            list[taskCount] = new Deadline(taskName, details);
+            break;
+        case "event":
+            list[taskCount] = new Event(taskName, details);
+            break;
+        }
+
+        taskCount++;
+        String plural = (taskCount - taskCompleted) == 1 ? "" : "s";
+        printLine();
+        System.out.println("      Fine. Added to your list:");
+        System.out.println("      " + list[taskCount - 1]);
+        System.out.println("      You have " + (taskCount - taskCompleted) + " pending task" + plural + ". aWeSoMe!!1!1!!");
+        printLine();
+    }
+
+    /**
+     * Marks tasks in list[] as done, if they exist or has not been completed.
+     *
+     * @param taskNumber Number n associated with the task, where n is the nth task in list.
+     */
+    public static void crossOutTask(int taskNumber) {
+        printLine();
+        if (taskNumber < 0 || taskNumber >= taskCount) {
+            System.out.println("      Wha- Hey! Task does not exist!");
+        } else if (list[taskNumber].isDone) {
+            System.out.println("      Dude... the task has already been done.");
+        } else {
+            taskCompleted++;
+            list[taskNumber].isDone = true;
+            System.out.println("      I've marked this task as done:");
+            System.out.println("        [" + list[taskNumber].getStatusIcon() + "]" + list[taskNumber].description);
+        }
+        printLine();
+    }
+
+    /**
+     * Engages user base on what the user has typed and executes that command.
+     */
+    public static void engageUser() {
+        Scanner text = new Scanner(System.in);
+        String taskType;
+        String userInput;
+        String[] parts;
+
+        String taskName = "";
+        String details = "";
+        boolean isUsing = true;
+
+        do {
+
+            taskType = text.next();
+
+            switch (taskType) {
+            case "bye":
+                isUsing = false;
+                break;
+            case "hello":
+            case "hi":
+            case "yo":
```

Comment:
haha, its great that the bot is customised and has these additional "easter eggs"